### ISSUE TITLE
[Feature Request] Global shortcuts

### ISSUE THREAD

**Issue by AlexeyKot** [KryDos/todoist-linux#4]
Hey! 
It would be great to have a global shortcut to show Todoist window, something like "Alt+Shift+T". It would be cool to make it customizable at least editing some file though.

Extra bonus
It would awesome to have global shortcut to add a new note. I know that it is possible, but I also know that this task is more tricky.

You may ask [Klaus](https://github.com/klauscfhq) how to achieve that, because he has created unofficial Evernote client app in Electron - [Tusk](https://github.com/klauscfhq/tusk) and implemented such feature.

**Comment by KryDos** [comment]
Hey! thanks for the great ideas!

`It would awesome to have global shortcut to add a new note` - could you please describe a bit this idea? As far as I know there is no "Notes" in Todoist in general, you can attach comments (which are notes, yeah) to a task. The shortcust for this is `CTRL+m`. It's shortcut of Todoist itself (their web version). Is it what you need?

**Comment by KryDos** [comment]
As for "Show todoist" shortcut, I'll make it ASAP. Try to make it configurable.

**Comment by AlexeyKot** [comment]
@KryDos sorry, I meant to create a global shortcut to create a new task, not note :-)

**Comment by KryDos** [comment]
@AlexeyKot but it's exists already :) it's actually the only global shortcut the app has. 

`CTRL+ALT+a` - calls "quick task" functionality of todoist.

**Comment by roock** [comment]
I'm having issues with the global shortcuts. Ctrl+Alt+t works, the others don't work (not even inside the application). Any suggestions/hints?

**Comment by KryDos** [comment]
hey @roock, 
I saw similar issue on Gnome 3. It was because shortcut handler of gnome has much more priority than the app's one. 

I'm not sure if you're on Gnome though but it's pretty possible that other desktop envs behaves the same. Could you please check your global shortcuts of the system and make sure there are no conflicts between  system and app shortcuts. 

And could you please share your distro name and what desktop environment do you use.

**Comment by roock** [comment]
@KryDos Thank you for the tip. The shortcuts were indeed already assigned in my desktop environment (KDE on Ubunut 18.04). All shortcuts now work for me.
btw: did you think about making the shortcuts configurable?

**Comment by KryDos** [comment]
@roock yeah, this is definitely something I was thinking about and want to implement. 
I don't think it will be separated page with GUI interface where you can change it. But it will be a file you can edit to change the shortcuts (and other settings). 

I've just CC'd you in the issue I've just created. I'll let you know there when it's done.
